airbyte-integrations/connectors/source-tempo/source_tempo/tempo.yaml

```diff
@@ -0,0 +1,101 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: ["results"]
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    url_base: "https://api.tempo.io/4/"
+    http_method: "GET"
+    error_handler:
+      type: CompositeErrorHandler
+      # ignore 403 error but retry default retriable http errors (429, 500 - 600)
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - http_codes: [403]
+              action: IGNORE
+        - type: DefaultErrorHandler
```

Comment:
it is ok that `- type: DefaultErrorHandler` appears twice?

Reply:
yep, that's done on purpose: first one processes only http status 400 responses, the second one - all others